### ISSUE TITLE
editor.multiCursorModifier is too restrictive for users on Kubuntu as ALT is taken over by KWIN.

### ISSUE THREAD

**Issue by joshbenham** [Microsoft/vscode#41329]
- VSCode Version: 1.19.1
- OS Version: Kubuntu 17.10

Unfortunately when it comes to **editor.multiCursorModifier** as well as click to go through to definition there is a slight but on Kubuntu.

The problem occurs that you cannot exactly use both. 

Say if I wanted to have ctrl + click to go to definition which most editors use that would be fine. But it means that the only other option for **editor.multiCursorModifier** is ALT.

The problem here is that ALT in Kubuntu is taken over by KWin and it is used to move a window. You can click anywhere on the application and not just on the title bar. So these two functionalities are used in conjunction even though CTRL is really the only option.

Is there a way to set the go to definition to "shift + click" ? Or "shift + ctrlCmd" as it is rather limited. 

Steps to Reproduce:

1. Install Kubuntu
2. Try ALT and Click on a definition to see if you can click through

Does this issue occur when all extensions are disabled?: Yes/No

Yes.

**Comment by joshbenham** [comment]
Just to let you know that I have overcome this problem through KDE by doing

`System Settings -> Windows Behaviour -> Windows Actions` and setting the `Modifier Key` to be Meta (windows key) instead of ALT.

Although for most users this may not be a good approach...

**Comment by Troupal** [comment]
I agree with @joshbenham, the option Shift+left click, Shift+right click (as in sublime-text) or alt+maj+left click is really (in my opinion) better than the alternative you proposed or the alternative that VSCode proposes (ctrl+left click).
So I'm not an expert on VSCode, but would there be a way to change that, and if so, how?
